### PR TITLE
Fixed a bug where nodes could be clicked during scroll animation

### PR DESCRIPTION
**Pull request by srdjanm88**
@justinthec these are some pretty spicy memes check it out fam :100:  :ok_hand:  :fire:

### CODE REVIEW

File: redditnav.js [hunk 1]
```diff
@@ -61,6 +61,8 @@ function getNextParent(direction, parentComments) {
 }
 
 function goToNextParent(direction) {
+  if(scrolling)
```
Comment (justinthec) [inline redditnav.js]: put a space between if and (

**Comment by justinthec** [comment]
You can remove the
```js
   if (scrolling)
     return;
```
from line 11.

**Comment by srdjanm88** [comment]
@justinthec how about now :^)

**Comment by justinthec** [comment]
🚢

**Pull request merged**